lc0_main.go

```diff
@@ -242,9 +271,16 @@ func (c *cmdWrapper) launch(networkPath string, args []string, input bool) {
 			}
 		}
 		c.Cmd.Args = append(c.Cmd.Args, fmt.Sprintf("--backend-opts=%s", *backopts))
+	} else if hasCudnnFp16 {
```

Comment:
Needs to have a hasCudnn path after this - otherwise gpu target will be ignored for cudnn.

Reply:
If cudnn-fp16 fails the error handler hardcodes backend=cudnn in backopts. 
However it is a good idea and cleaner to just set hasCudnnFp16 to false in the error handler.

Reply:
I guess I was talking about a hypothetical build, built without fp16 support.